lib/screens/v2/candidate_generator/widgets/departures.ex

```diff
@@ -176,6 +186,11 @@ defmodule Screens.V2.CandidateGenerator.Widgets.Departures do
     end
   end
 
+  def fetch_section_departures(_, _, _, _) do
+    # Only get to this pattern match if wayfinding_only_text is not nil, in which case, we fetch no departures
+    :wayfinding_only
```

Comment:
suggestion: This could be `{:ok, []}` and then we would not need the extra clause of `normal_section_rows`.